README.md

```diff
@@ -200,8 +201,8 @@ with the Elasticsearch Monitoring team.
 
 The additional parameters:
   * `--id` _REQUIRED_ &nbsp;&nbsp;&nbsp;  The cluster_id of the cluster you wish to retrieve data for. Because multiple clusters may be monitored this is necessary to retrieve the correct subset of data. If you are not sure, see the --list option example below to see which clusters are available.
-  * `--startDate`  &nbsp;&nbsp;&nbsp;  Date for the earliest day to be extracted. Defaults to the date the utility was run, in UTC, minus the current interval value. Must be in the format yyyy-MM-dd.
-  * `--startTime` &nbsp;&nbsp;&nbsp;  The clock time of that date when the requested statistics begin. Defaults to the time the utility was run in UTC. Must be in the format HH:mm 24 hour format.
+  * `--startDate`  &nbsp;&nbsp;&nbsp;  Date for the earliest day to be extracted. Defaults to the current date minus the default interval in UTC. Must be in the format yyyy-MM-dd.
+  * `--startTime` &nbsp;&nbsp;&nbsp;  The clock time of that date when the requested statistics begin. Defaults to current time in UTC. Must be in the format HH:mm 24 hour format.
```

Comment:
I had already started looking into this during the Support Meeting because it just didn't feel right. Was thinking about Demo Day and how I'd hate to have to explain this. I didn't delete it since I assumed there would be a delay while you were looking at the analyzer. So I just pushed the changes to this branch. 

So first I eliminated the time as a separate parameter. This means you have to put the value in quotes because of the space but it makes it easier overall, I think.

There's already a spot in there if you look where i do that explicit ZoneDateTime force to UTC - it's just that I'm using the "+0" instead of "UTC". And I had neglected to put that in the other ones. I didn't want to make it global to the Java process since that could muck with logging. So I wanted to keep it encapsulated here. 

The instructions now indicate to just enter the time in the same TZ as local, or whatever the machine they are running for it is set for, and then it's converted to UTC when I generate the statement parameters. I think this is easier to grasp. And it lets you run with the defaults and just get the last 6 hours of data with no user intervention.

Have also adjusted the tests and readme to reflect this.